files/image_config/pcie-check/pcie-check.sh

```diff
@@ -16,11 +16,12 @@ function debug()
 
 function check_and_rescan_pcie_devices()
 {
-    PCIE_CHK_CMD='sudo pcieutil check | grep "$RESULTS"'
+    PCIE_CHK_CMD='sudo pcieutil check |grep "$RESULTS"'
```

Comment:
Why remove the space?

Reply:
I think this came from my previous commits while merging master branch. I will add it back.